packages/react-native/src/primitives/Button/tokens.ts

```diff
@@ -0,0 +1,5 @@
+import { ButtonStyles } from './types';
```

Comment:
Why do we need this file?

Reply:
For the defaultTheme, gets merged and cleaned up in the `createReactNativeTheme` function.